components/Bounty/BountyAlert.tsx

```diff
@@ -8,9 +8,14 @@ import numeral from "numeral";
 type BountyAlertParams = {
   bounty: Bounty;
   allBounties: [Bounty];
+  bountyType: string;
```

Comment:
I spent a bunch of time curating the `Bounty` class into an object that contains functionality and data related for all things bounty.

I think it would be really super to add an attribute called "type" which can be "question" or "bounty". This type doesn't have to be a BE thing, just a FE thing to make things easier to use.